### DISCUSSION TITLE
Connecting Pythagora with DeepSeek API

### DISCUSSION THREAD

**Discussion by tarek101** [Pythagora-io/gpt-pilot#929] [Q&A]
Can I connect Pythagora with DeepSeek API, instead of OpenAI API? Deepseek API is compatible with OpenAI and way cheaper. If it is possible, how can I specify the model to use, i.e., DeepSeek-Coder-Instruct-33B model ?

**Comment by Simpwell** [comment]
There is fine-tuned DeepSeek coder specialized for Pythagora. 

https://github.com/MoonlightByte/Pythagora-LLM-Proxy
https://huggingface.co/LoupGarou/deepseek-coder-33b-instruct-pythagora-v3-gguf
https://huggingface.co/LoupGarou/deepseek-coder-6.7b-instruct-pythagora-v3-gguf
